pkg/synopsysctl/cmd_delete.go

```diff
@@ -132,6 +132,28 @@ var deleteOpsSightCmd = &cobra.Command{
 	},
 }
 
+// deletePolarisCmd deletes Polaris instances from the cluster
+var deletePolarisCmd = &cobra.Command{
+	Use:           "polaris NAME...",
+	Example:       "synopsysctl delete polaris <name>\nsynopsysctl delete polaris <name1> <name2> <name3>",
+	Short:         "Delete one or many polaris instances",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 0 {
```

Comment:
The "Use" above implies this command takes in one or more arguments.

Reply:
fixed